llvm/test/tools/llvm-reduce/operands-to-args.ll

```diff
@@ -96,7 +96,7 @@ declare void @ptr_user(ptr)
 
 ; INTERESTING-LABEL: define void @calls_and_passes_func(
 ; REDUCED-LABEL: define void @calls_and_passes_func(ptr %has_callee_and_arg_user) {
-; REDUCED: call void @has_callee_and_arg_user(ptr %has_callee_and_arg_user)
+; REDUCED: call void @has_callee_and_arg_user(ptr %has_callee_and_arg_user, ptr null)
```

Comment:
why do we pass `null` here now, should we not ideally introduce a new argument?

Reply:
Whenever this this does anything, the call sites are adjusted to pass the default argument to the new argument